tests/migrations/test_operations.py

```diff
@@ -1937,6 +1937,44 @@ def test_remove_field(self):
         self.assertEqual(definition[1], [])
         self.assertEqual(definition[2], {"model_name": "Pony", "name": "pink"})
 
+    def test_remove_field_non_nullable(self):
```

Comment:
I would remove most of the comments except the MySQL one or each need to end with a full stop.